File: 30 days of code/__ C#/Day 06 - Let's review.cs
/**
 *              LET'S REVIEW | C#
 * Domain       30 DAYS OF CODE
 * Sub domain   DAY 06
 */

class Program
{
    //static TextReader read = Console.In;
    static StreamReader read = new StreamReader("input.txt");

    static void Main(string[] args)
    {
        int testCases = int.Parse(read.ReadLine());
        while (testCases-- > 0)
        {
            string input = read.ReadLine();
            LetsReview(input);
        }
        read.Close();
    }

    private static void LetsReview(string input)
    {
        string even = string.Empty;
        string odd = string.Empty;

        for (int i = 0; i < input.Length; ++i)
        {
            if (i % 2 == 0)
            {
                even += input[i];
            }
            else
            {
                odd += input[i];
            }
        }

        Console.WriteLine($"{even} {odd}");
    }
}

File: 30 days of code/__ C++/Day 06 - Let's review.cpp
/**
*              LET'S REVIEW | C++
* Domain       30 DAYS OF CODE
* Sub domain   DAY 06
*/


#include "stdafx.h"
#include <iostream>
#include <fstream>
#include <string>

using namespace std;

ifstream file("input.txt");
istream& f = file;

void letsReview(string& input) {
	string even = "";
	string odd = "";

	for (int i = 0; i < input.size(); ++i) {
		if (i % 2 == 0) {
			even.append(1, input[i]);
		}
		else {
			odd.append(1, input[i]);
		}
	}

	cout << even << " " << odd << endl;
}

void main() {
	int testCases; f >> testCases;
	while (testCases-- > 0) {
		string input;
		getline(f >> ws, input);
		letsReview(input);
	}
}